lib/trailblazer/operation.rb

```diff
@@ -45,6 +45,18 @@ def present(*params)
       def contract(&block)
         contract_class.class_eval(&block)
       end
+      
+      def collection(*params, &block)
```

Comment:
I like the idea of this, and the tests. :kiss: 

However, there's no need to introduce a new API. What if we just add an `Operation::Collection` module that overrides a few methods?

`#model!` and `#setup_model!` are exactly designed to do what your `#fetch` does. This way, we can use `Op::present` without changing a single line of code (maybe skip creating the form object, though).

Not sure if we even need the `Controller#collection` method - it shouldn't matter to the invoker whether you're dealing with a collection or a model.